examples/cookbook/package.json

```diff
@@ -12,6 +12,7 @@
     "chalk": "^4.1.1",
     "homedir": "^0.6.0",
```

Comment:
Not related _directly_  to this PR, but just something that I happened to notice since you aren't using it in this example -- can we delete this package from our dependencies entirely since it is part of the node `os` package implementation? I don't think we even use it, so it'd be good to get it out of here as a follow-up PR

Reply:
Will do!